azurerm/internal/services/synapse/synapse_private_link_hub_test.go

```diff
@@ -0,0 +1,116 @@
+package synapse_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance/check"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/synapse/parse"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/tf/pluginsdk"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+type SynapsePrivateLinkHubResource struct{}
+
+func TestAccSynapsePrivateLinkHub_create(t *testing.T) {
```

Comment:
We prefer to align the test name and the config method name where there's a 1:1 correspondence

```suggestion
func TestAccSynapsePrivateLinkHub_basic(t *testing.T) {
```